### PR TITLE
Adding moi static files to the gitignore

### PR DESCRIPTION
**Pull request by josenavas**
I almost add these files by mistake. Adding them to the gitignore will help.

fast to review....

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,3 +57,7 @@ gg_13_8-*
 
 # sphinx documentation
 qiita_pet/static/doc/
+
+# moi static files
+qiita_pet/static/vendor/js/moi.js
+qiita_pet/static/vendor/js/moi_list.js
```
Comment (wasade) [inline .gitignore]: possible to add a newline?

**Comment by wasade** [comment]
Good catch...

File: .gitignore (hunk 1 above)
Comment (josenavas) [inline .gitignore]: Done!

**Comment by antgonza** [comment]
:+1:

**Comment by josenavas** [comment]
@wasade @antgonza wanna merge?

**Pull request merged**